app/src/main/java/com/sunasterisk/a14day_challenge/adapter/DetailExercisesAdapter.kt

```diff
@@ -0,0 +1,58 @@
+package com.sunasterisk.a14day_challenge.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.sunasterisk.a14day_challenge.R
+import com.sunasterisk.a14day_challenge.data.model.DetailExercise
+import kotlinx.android.synthetic.main.recycler_view_list_exercises.view.*
+
+class DetailExercisesAdapter(
+    private val exercises: List<DetailExercise>,
+    private val onExerciseListener: OnExerciseListener
```

Comment:
```suggestion
    private val onExerciseClick: (position: Int) -> Unit
```
đã dùng Higher-order function rồi thì không cần interface kia đâu